README.md

```diff
@@ -94,7 +94,7 @@ constructors. `case` can be used as a control structure for handling the
 different values a type can have:
 
 ```javascript
-var Move = Type({Up: [], Right: [], Down: [], Left: []});
+var Move = Type({Up: [], Right: [], Down: [], Left: [], Jump: [], Fire: [Number]});
```

Comment:
What about renaming `Move` to `Action`? Given these two additions that sounds like a better name to me.